app/controllers/rentals_controller.rb

```diff
@@ -0,0 +1,52 @@
+class RentalsController < ApplicationController
+  def checkout
```

Comment:
Nothing to ensure that there are enough movies in the inventory to check it out?